### ISSUE TITLE
Active tab highlight

### ISSUE THREAD

**Issue by Gabrieleiro** [minbrowser/min#703]
Highlight color(s) for active tabs should blend with the current page's main color. 

The way it is now is counter intuitive and it leads you to think inactive tabs are the ones you're looking at.

**Comment by PalmerAL** [comment]
That's an interesting idea, it does seem to help make it somewhat easier to see which tab is selected (example: [before](https://user-images.githubusercontent.com/10314059/51722257-6e1acf00-201a-11e9-957b-a5525e007f00.png) vs [after](https://user-images.githubusercontent.com/10314059/51722267-75da7380-201a-11e9-9c88-c5fa1ba3f3cc.png)). However, I think it does have a few downsides:

* If the majority of the tab bar is either much lighter or darker than the accent color (which we would need to do to provide contrast for the current tab), it makes the tab bar look kind of washed out.
* A lot of sites have a fixed header that's the same color as their theme color. When most of the tab bar is this same color, it creates the appearance of everything being a single bar, which reduces clutter, but the effect doesn't work very well when most of the tab bar is the accent color instead.

It's definitely something to consider, though.

**Comment by Tomadelostacos** [comment]
I personally use a bold font for the active tab. I didn't commit anything, but it's a line i can commit if you think the idea is good

**Comment by PalmerAL** [comment]
That sounds like a good idea, I'll try it for a few weeks and see how well it works.

**Comment by PalmerAL** [comment]
@Tomadelostacos I've been testing this with `font-weight: 500`, and I think it does make it easier to see the active tab, which is nice! However, changing the font weight also makes the width of the text change as you switch tabs, which is really annoying. To fix this, I think we'd have to adjust the letter spacing on bold text so that it ends up being the same width; there's a couple ways we could do this:

1. Hard-code a bold letter-spacing for each [supported font](https://github.com/minbrowser/min/blob/master/css/base.css#L6), and just let the letter spacing be incorrect if we fall back to the generic `sans-serif` font.
2. Compute the correct letter-spacing on startup by putting some text in a container, measuring its width, and progressively shrinking its letter-spacing until it matches the width of non-bolded text. This would work regardless of the font, but it seems fairly complicated, and might also be slow (although maybe we could cache it?)

There's also an interesting difference depending on the background color - with white text on a color background, the bolding seems fairly reasonable, but with black text on a white background, it seems too bold. I think we can fix this by switching to a dark-gray text color instead of pure black (which is something we should probably do anyway).

**Comment by Tomadelostacos** [comment]
Nice ! I have a pretty decent feel when just setting the letter-spacing of an active bold title to 0, but i think we can do better than that. 
I agree on the black text on a white background, it surely looks way better with a #333 or something like that.

**Comment by PalmerAL** [comment]
I tried the second approach I mentioned ([code](https://gist.github.com/PalmerAL/ef72bb76f08d191499e72d2facd9217e)), but it ended up being fairly slow and not actually working that well (I think because the width also depends on the number of spaces vs letter characters), so the hard-coded approach is probably better.

**Comment by ronilaukkarinen** [comment]
> The way it is now is counter intuitive and it leads you to think inactive tabs are the ones you're looking at.

Exactly. I always tend to click the lighter ones as usually darker indicate the active one.

So +1 to improving this...
